Formula/j/jpeg-xl.rb

```diff
@@ -50,16 +51,25 @@ class JpegXl < Formula
     url "https://github.com/webmproject/sjpeg.git",
         revision: "e5ab13008bb214deb66d5f3e17ca2f8dbff150bf"
   end
+  resource "jpeg-turbo" do
```

Comment:
There is already a formula for this

Reply:
It uses *source* from jpeg-turbo, not its delivered header. Specifically a header.in file that gets filled at compile-time.